project/script_IR.tcl

```diff
@@ -27,10 +27,10 @@ create_clock -period 240MHz -name slow_clock
 create_clock -period 360MHz -name fast_clock
 
 set nProc [exec nproc]
-csim_design -mflags "-j$nProc" -argv "--link,8 --dtcSplit,0 --tkNonant,4"
+csim_design -mflags "-j$nProc" -argv "--link,8 --dtcSplit,0 --tkNonant,4 --lastBx,100"
```

Comment:
@sseifeln Does the addition of this argument do anything, since the test bench is processing 100 events by default?